package.json

```diff
@@ -103,7 +103,7 @@
     "@vanilla-extract/webpack-plugin": "^2.1.11",
     "babel-plugin-istanbul": "^6.1.1",
     "cypress": "10.3.1",
-    "cypress-hardhat": "^1.0.1",
+    "cypress-hardhat": "^2.0.0",
```

Comment:
did we 1.0 too early here?

Reply:
We just had a breaking change, so we bumped the major. I'd consider this working-as-intended, I don't think we should assign too much value to "hitting 1.0".